### PR TITLE
Update .NET Core runtime version

### PR DESCRIPTION
**Pull request by alexzautke**

### CODE REVIEW

File: vonk/start.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ For non Windows systems, or if you want to use Docker for Windows, please look a
     }
 
 .. important:: the next step assumes you have a .Net Core environment installed. If not, please `download and install
-  <https://www.microsoft.com/net/download/core#/runtime/>`_ **.NET Core Runtime 2.0.0** before you continue.
+  <https://www.microsoft.com/net/download/core#/runtime/>`_ **.NET Core Runtime 2.1.10 or higher** before you continue.
```
Comment (cknaap) [inline vonk/start.rst]: We've arrived at 2.2. Would change to 'latest 2.2.x version'

Comment (alexzautke) [reply vonk/start.rst]: 2.1 is the LTS version. I expect that larger organisations will be stuck to 2.1 for a while. Should we really force the upgrade?

Comment (cknaap) [reply vonk/start.rst]: We already did upgrade. Your comment is right, but too late I'm afraid.

Comment (alexzautke) [reply vonk/start.rst]: Oh, right. I was still thinking about the last Vonk version not about the new one.

**Comment by cknaap** [comment]
Integrated this change in the 2.0.0 release notes update.

**Pull request closed without merging**